notebooks/Advanced_Usage.ipynb

```diff
@@ -37,7 +37,11 @@
     "  * [Accessing callback parameters](#Accessing-callback-parameters)\n",
```

Comment:
<p>Maybe instead of writing 'would raise an error', quote the actual error (makes it searchable).</p><p><br></p>

 _Reply via <a href='https://app.reviewnb.com/skorch-dev/skorch/pull/457/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='84'/>

Reply:
The error would be

`AttributeError: 'tuple' object has no attribute 'size'`

It is raised within pytorch code. We could catch that but this seems overly specific. We could think of another strategy to deal with that situation, but that would be out of scope for this PR.

Reply:
Yeah I just thought that quoting the error in the notebook would make it searchable through google if someone searches for `skorch AttributeError has not attribute size` or something similar.